raco/backends/myria/myria.py

```diff
@@ -920,10 +920,12 @@ class MyriaQueryScan(algebra.ZeroaryOperator, MyriaOperator):
 
     """A Myria Query Scan"""
 
-    def __init__(self, sql, scheme, num_tuples=algebra.DEFAULT_CARDINALITY,
+    def __init__(self, sql, scheme, source_relation_keys,
+                 num_tuples=algebra.DEFAULT_CARDINALITY,
```

Comment:
Actually, I copied that from the `add_broadcast` branch. I put this in a separate PR because the changes were self-contained WRT broadcast, but maybe you should just review that branch instead and get all the changes at once. I'll close the existing RACO broadcast PR (since it's on an obsolete branch) and open a new one.